app/dialogs.js

```diff
@@ -26,12 +27,14 @@ function getWindow (win) {
 
 var appDialogs = {
 
+    localizer: new Localizer(app.getLocale()),
```

Comment:
I don't find storing Localizer here very logical, but we can keep it as is and change it later.